src/pages/registration/resource_type_tab/SelectRegistrationTypeField.tsx

```diff
@@ -90,10 +97,19 @@ export const SelectRegistrationTypeField = () => {
               },
               false
             );
-          setFieldValue(instanceTypeBaseFieldName, { ...emptyBookPublicationInstance, type: newInstanceType }, false);
+            setFieldValue(instanceTypeBaseFieldName, { ...emptyBookPublicationInstance, type: newInstanceType }, false);
+          } else {
+            // If we move between different categories in the same group with the same fields, we keep the info in the fields
```

Comment:
Denne kommentaren gjentar seg mange ganger nedover, holder det med èn øverst? Behøver man kommentaren?

Reply:
Agreed. But keeping it for the first case.